src/s25update.cpp

```diff
@@ -13,7 +13,16 @@
 #include <algorithm>
 #include <array>
 #include <bzlib.h>
+
 #include <curl/curl.h>
+/*
+ * CURL_AT_LEAST_VERSION took out from libCURL 7.43+ of curl/curlver.h for easier readability.
+ */
+#ifndef CURL_AT_LEAST_VERSION
```

Comment:
I'd put that right in the define as otherwise one could think the `#ifndef` is part of what was taken.
Also the comment can be shorter and on a single line, like this:
```suggestion
#ifndef CURL_AT_LEAST_VERSION
// taken from curl/curlver.h of  libCURL 7.43+ for easier readability.
```

Reply:
Sure

Reply:
Merging as squashed commit?